src/Chapter1.hs

```diff
@@ -635,9 +650,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n
+    | (abs n) > 9 = firstDigit (div (abs n) 10)
+    | otherwise = abs n
```

Comment:
You did it! Cool 😎 

You can notice that `abs n` is used several times in the body. That usually indicates that you can have a helper function for that. In this case, you could move it in the `where` clause 🙂